### ISSUE TITLE
stream_monitor.py ignores log content

### ISSUE THREAD

**Issue by msjonker** [parkpow/deep-license-plate-recognition#210]
Currently, stream_monitor.py just parses the log lines and looks to see if there are recent logs for cameras, regardless of what the log messages actually are. There are times where the log for a camera is actually an error, such as:
```
INFO:ingress-north:2024-07-16T13:31:45+00:00: URL unreachable. Will attempt to reconnect in 403.2 seconds
```
In this cases such as this, stream_monitor will return that the camera is running, when it is not.

As I've stated before, it would be great if this monitor was part of the Stream image itself and did not rely on having to parse docker logs and end up with issues like this, as well as have to spin up a separate container just to run this.

**Comment by adolfoarmas** [comment]
@msjonker does `/status/` endpoint work for you instead of `stream_monitor` script?
See details in the documentation: https://guides.platerecognizer.com/docs/stream/monitoring/#monitor-using-heartbeat-server

It is available from version 1.51.0

**Comment by msjonker** [comment]
We don't have 1.51.0+ running at our sites yet, but this looks like exactly what I'm looking for! Thanks @adolfoarmas!
